dev-util/cepl/cepl-8.3.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+DESCRIPTION="An interactive C and C++ read–eval–print loop."
+HOMEPAGE="https://github.com/alyptik/cepl"
+SRC_URI="https://github.com/alyptik/cepl/releases/download/v${PV}/cepl-${PV}.tar.gz"
+S="${WORKDIR}"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+RDEPEND="
+	sys-libs/readline
+	dev-libs/libelf
+	sys-libs/ncurses
```

Comment:
```suggestion
	sys-libs/readline:=
	virtual/libelf:=
```
Since cepl compiles just fine without the ncurses include directives, I'd drop it from the dependencies.

https://github.com/alyptik/cepl/issues/7